rllib/algorithms/algorithm_config.py

```diff
@@ -3648,12 +3664,30 @@ def _validate_evaluation_settings(self):
             )
 
         # If async evaluation is enabled, custom_eval_functions are not allowed.
-        if self.enable_async_evaluation and self.custom_evaluation_function:
+        if (
```

Comment:
Can we update the comment above to mention that this is only NOT allowed, iff we are using the old RolloutWorkers?